src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAWindowsDesktopProject.cs

```diff
@@ -342,6 +345,60 @@ public void SupportedOSPlatformVersionCanBeSetInDirectoryBuildTargets()
         }
 
 
+        [WindowsOnlyTheory]
+        //  Basic Windows TargetFramework
+        [InlineData("net5.0-windows10.0.19041.0", false, null, "10.0.19041.*")]
+        //  Basic UseWindowsSdkPreview usage
+        [InlineData("net5.0-windows10.0.99999.0", true, null, "10.0.99999-preview")]
```

Comment:
Do these packages ever use full semantic versioning with build numbers and metadata? It sounded from the comments in the build task that the patterns are fixed, e.g. you add "preview" and that will never change.

Reply:
Yes, the release versions of the packages use 4 digit build numbers, but NuGet doesn't allow you to use prerelease information in a package version if it has 4 numbers.  So the preview packages will only have 3 numbers, and they plan to always just have them marked as `-preview`.  If they need something else, then the `WindowsSdkPackageVersion` property can be used to explicitly specify the full package version.